packages/@ama-sdk/schematics/package.json

```diff
@@ -43,6 +40,7 @@
     "@angular-devkit/core": "^16.0.5",
     "@angular-devkit/schematics": "^16.0.5",
     "@angular-devkit/schematics-cli": "^16.0.5",
+    "@o3r/schematics": "workspace:^",
```

Comment:
Should not depend on `@o3r/schematics` which brings Angular.
This actually the reason to split @o3r/schematics